examples/nlp/text_classification/conf/text_classification_config.yaml

```diff
@@ -23,44 +23,42 @@ trainer:
   max_steps: null # precedence over max_epochs
   accumulate_grad_batches: 1 # accumulates grads every k batches
   amp_level: O0 # O1/O2 for mixed precision
+  precision: 32 # Should be set to 16 for O1 and O2
   distributed_backend: ddp
   checkpoint_callback: False  # Provided by exp_manager
   logger: False  # Provided by exp_manager
+  resume_from_checkpoint: null # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step, LR schedulers, apex, etc.
 
 model:
-  data_dir: ??? # /path/to/data
-  class_balancing: null # or weighted_loss
+  dataset:
+    do_lower_case: false # true for uncased models, false for cased models, will be set automatically if pre-trained tokenizer model is used
+    max_seq_length: 36
+    class_balancing: null # or weighted_loss
+    use_cache: false
+    num_workers: 2 # number of workers for data loaders
+    drop_last: false # drops the last last batch if it is smaller than the batch size
+    pin_memory: false # enables pin_memory feature of the data loaders
 
   train_ds:
-    prefix: train
+    file_name: ???
```

Comment:
please put null

Reply:
Shouldn't it be ??? to be necessary? Can user ignore training file and pass None?

Reply:
can we use just for eval if train_file is null?